hack/generate-yamls.sh

```diff
@@ -118,3 +121,5 @@ echo "All manifests generated"
 ls -1 ${SERVING_YAML} > ${YAML_LIST_FILE}
 ls -1 ${YAML_OUTPUT_DIR}/*.yaml | grep -v ${SERVING_YAML} >> ${YAML_LIST_FILE}
 ls -1 ${ISTIO_CRD_YAML} ${ISTIO_YAML} ${ISTIO_LEAN_YAML} ${ISTIO_KNATIVE_EXTRAS_YAML} >> ${YAML_LIST_FILE}
+ls -1 ${CERT_MANAGER_CRD_YAML} >> ${YAML_LIST_FILE}
```

Comment:
We should not ship `cert-manager`.  We will need to outline in our `docs` how to install `cert-manager` as part of setup (or avoid the cert-manager install)